src/kubernetes.cc

```diff
@@ -115,6 +117,50 @@ KubernetesReader::KubernetesReader(const Configuration& config,
                                    HealthChecker* health_checker)
     : config_(config), environment_(config), health_checker_(health_checker) {}
 
+MetadataStore::Metadata KubernetesReader::GetMetadataOnly(
+    const json::Object* resource, Timestamp collected_at, bool is_deleted) const
+    throw(json::Exception) {
+#ifndef ENABLE_KUBERNETES_METADATA
```

Comment:
Do you want to add a commit to get rid of this macro? I don't think it serves any purpose anymore. We could even get rid of it on `master`...

Reply:
done